### PR TITLE
Fix some errors in the current integration tests and update the docum…

### PR DESCRIPTION
**Pull request by sarthak-kaps**
…entation on how to run the integration test suite.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -80,20 +88,49 @@ where
...
+  NOTE: The Stored Credential of the user must be kept in a file called StoredCredential, and we only need to pass the path of the folder containing the file.
+
+- `api.test.authInfoUser3`: Authentication information for User3. This consists of the email id of User3, the path to the folder containing the user's Stored Credentials and the path to the JSON file containing client secret for desktop app.\
+  NOTE: The Stored Credential of the user must be kept in a file called StoredCredential, and we only need to pass the path of the folder containing the file.
+
+- `api.test.authInfo`: Authentication information for Admin. This consists of the email id of Admin, the path to the folder containing the admin's Stored Credentials and the path to the JSON file containing client secret for desktop app.\
+  NOTE: The Stored Credential of the admin must be kept in a file called StoredCredential, and we only need to pass the path of the folder containing the file.
+
+- `api.test.groupPrefix` : The name of a group created for testing purpose. This test must contain User3 and should not contain User1 and User2.
```
Comment (emailofanuj) [inline README.md]: Modify the comment to "This group must contain User3 and should not contain User1 and User2."

File: README.md [hunk 2]
```diff
@@ -80,20 +88,49 @@ where
 - `api.test.identity.domain`: the domain associated with a Cloud Identity customer. Temporary users
```
Comment (emailofanuj) [inline README.md]: modify test.identity to test only in different arguments like api.test.identity.identitySourceId, etc.

File: README.md [hunk 3]
```diff
@@ -80,20 +88,49 @@ where
...
   will be created on this domain during the execution of the test.
 
+- `api.test.adminUserId`: Email ID of admin account.
+
+- `api.test.rootUrl`: the base url of api endpoint. For prod the `rootUrl` is https://cloudsearch.googleapis.com/. For staging the `rootUrl` is https://staging-cloudsearch.sandbox.googleapis.com/.
+
```
Comment (emailofanuj) [inline README.md:94]: api.test.searchApplicationId definition is missing here

File: README.md [hunk 4]
```diff
@@ -80,20 +88,49 @@ where
...
 ```
-mvn verify \
+mvn integration-test \
     -DskipITs=false \
-    -Dapi.test.serviceAccountPrivateKeyFile=key.json \
-    -Dapi.test.sourceId=01cb7dfca117fbb591360a2b6e46912e \
-    -Dapi.test.identitySourceId=01cb7dfca117fbb591360a2a5b64177c \
-    -Dapi.test.customerId=C14e3bmn2 \
-    -Dapi.test.domain=company.com
+    -Dtest=FakeIndexingRepositoryIT \
```
Comment (emailofanuj) [inline README.md]: should we just drop this argument so that both identity source and indexing tests could be run?

File: README.md (hunk 1 above)
Comment (sarthak-kaps) [reply README.md]: Done. Thanks for pointing out.

File: README.md (hunk 2 above)
Comment (sarthak-kaps) [reply README.md]: Done.

File: README.md (hunk 4 above)
Comment (sarthak-kaps) [reply README.md]: Yeah, right. I was supposed to remove this. Thanks.

**Review by emailofanuj — APPROVED** [review]

**Pull request merged**